### PR TITLE
[12.0][ADD] portal_recaptcha

### PR DESCRIPTION
**Pull request by huguesdk**
move `portal_recaptcha` module from [coopiteasy/vertical-cooperative](https://github.com/coopiteasy/vertical-cooperative) to the oca.

### CODE REVIEW

File: portal_recaptcha/migrations/12.0.1.0.1/pre-migration.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+def migrate(cr, version):
```
Comment (robinkeunen) [inline portal_recaptcha/migrations/12.0.1.0.1/pre-migration.py]: Can you remove the migration scripts for older versions (eg not present on the OCA ?) ?

**Review by robinkeunen — APPROVED** [review]
The screenshot recaptcha_setting.png still refers to v9 and to company settings. Otherwise LGTM

**Review by carmenbianca — APPROVED** [review]
LGTM

**Comment by mathben** [comment]
Travis CI output : 
```
ERROR: Could not find a version that satisfies the requirement pylint-plugin-utils==0.7 (from -r /home/travis/maintainer-quality-tools/travis/pylint_odoo_requirements.txt (line 1)) (from versions: 0.1, 0.1.1, 0.2, 0.2.1, 0.2.2, 0.2.3, 0.2.4, 0.2.6, 0.3, 0.4, 0.5, 0.6)

ERROR: No matching distribution found for pylint-plugin-utils==0.7 (from -r /home/travis/maintainer-quality-tools/travis/pylint_odoo_requirements.txt (line 1))
```

**Comment by robinkeunen** [comment]
@huguesdk can you take a look at this ?

**Comment by huguesdk** [comment]
everything is explained in OCA/maintainer-quality-tools#712.

**Comment by huguesdk** [comment]
@OCA/website-maintainers could someone please re-open this and re-run the ci? it should work now that the repository template has been updated in the 12.0 branch (switching from travis to github actions).

**Pull request closed without merging**